scipy/stats/stats.py

```diff
@@ -213,6 +214,7 @@
            'rankdata', 'rvs_ratio_uniforms',
            'combine_pvalues', 'wasserstein_distance', 'energy_distance',
            'brunnermunzel', 'epps_singleton_2samp', 'cramervonmises',
+           'alexandergovern', 'AlexanderGovernConstantInputWarning']
```

Comment:
```suggestion
           'alexandergovern', 'AlexanderGovernConstantInputWarning',
```

Reply:
Thanks, made that mistake when "fixing" merge conflicts.